packages/tools/devtools/devtools-view/src/components/utility-components/ConceptCellLayout.tsx

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import React from "react";
+import { Label, TableCellLayout } from "@fluentui/react-components";
+import { InfoLabel } from "@fluentui/react-components/unstable";
+
+/**
+ * {@link ConceptCellLayout} input props.
+ */
+export type ConceptCellLayoutProps = React.PropsWithChildren<{
+	/**
+	 * Icon to display to the left of the label.
+	 */
+	icon: React.ReactElement;
+
+	/**
+	 * (Optional) If specified, will display an "info" badge to the right of the label.
+	 * When clicked, a tooltip will be displayed with the provided contents.
+	 */
+	infoTooltipContent?: React.ReactElement | string;
+}>;
+
+/**
+ * Helper component for rendering the contents of a Table Header Cell intended to represent some Fluid or Devtools
+ * concept.
```

Comment:
I think this is right? Or did you mean "cells on the label column"?
```suggestion
 * Helper component for rendering the contents of a Table Cell intended to represent some Fluid or Devtools concept.
```

Reply:
Ah, good catch. Originally, I had scoped this helper to header cells, but ultimately realized it was more generally useful. Just failed to update the comment.